<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstroProQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="birthDetailsGroup">
          <property name="title">
           <string>Birth Details</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameInput"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="dateLabel">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateInput">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="timeLabel">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTimeEdit" name="timeInput"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="placeLabel">
             <property name="text">
              <string>Place:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="placeLayout">
             <item>
              <widget class="QLineEdit" name="placeInput"/>
             </item>
             <item>
              <widget class="QPushButton" name="searchButton">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="latLabel">
             <property name="text">
              <string>Latitude:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="latInput" readonly="true"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lonLabel">
             <property name="text">
              <string>Longitude:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lonInput" readonly="true"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="ayanamsaLabel">
             <property name="text">
              <string>Ayanamsa:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="ayanamsaCombo">
             <item>
              <property name="text">
               <string>Lahiri</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Raman</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generateButton">
          <property name="text">
           <string>Generate Chart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="chartOptionsGroup">
          <property name="title">
           <string>Chart Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="chartStyleCombo">
             <item>
              <property name="text">
               <string>North Indian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>South Indian</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="showAspectsCheck">
             <property name="text">
              <string>Show Aspects</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="enableZoomCheck">
             <property name="text">
              <string>Enable Zoom/Pan</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="ChartWidget" name="chartWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="analysisTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="dashaTab">
           <attribute name="title">
            <string>Dasha Periods</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="dashaTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="strengthTab">
           <attribute name="title">
            <string>Planetary Strengths</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="strengthTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="yogaTab">
           <attribute name="title">
            <string>Yoga Combinations</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTableWidget" name="yogaTable"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export Chart...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header>chartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
